### ISSUE TITLE
Iris cannot find UDUNITS on Windows

### ISSUE THREAD

**Issue by ocefpaf** [SciTools/conda-recipes-scitools#115]
I am puzzled that this is happening again:

`OSError: [UT_OPEN_ARG] Failed to open UDUNITS-2 XML unit database : "No such file or directory"`

That happens in both the `ioos` and  `scitools` versions of iris.  I am not sure what is going on.  Maybe my WindowsVM is broken.  So I appreciate if someone else can test this and let me know if the package is working for them.

NB: I can see the `site.cfg`.  It seems to be correctly created and is pointing to the right place to find the UDUNITS-2 DLL.

**Comment by rsignell-usgs** [comment]
So if I just try `anaconda install iris` and then `import iris` I can test this?

**Comment by ocefpaf** [comment]
I need to know if it is a clean environment in case you already have `iris` installed.  A `conda info` and `conda list` output would help.

**Comment by ocefpaf** [comment]
OK I can confirm that this only happens when starting Python from another hard disk letter.  This is a Windows idiosyncrasy that I have no idea how to solve!  Closing this.

**Comment by ocefpaf** [comment]
A workaround it is to load iris first and then switch directories using the `os` module.

```python
import iris
import os
os.chdir('E:')
```

(See http://nbviewer.ipython.org/gist/ocefpaf/6b4fb5f5a019c0af0a08)

**Comment by rsignell-usgs** [comment]
Interesting.  I imagine this windows workaround is likely useful in other situations as well.

**Comment by ocefpaf** [comment]
Probably.

PS: To be honest I am very frustrate with that solution.  There must be an easier and more "Windows natural way" to solve this.

**Comment by pelson** [comment]
I think this is a bug we need to fix so I'll re-open. Hopefully it will be a simple one...

**Comment by ColinTalbert** [comment]
Yeah this is biting me too.  A  couple of things I noticed (win64 anaconda), 

first my xml database is not in the path provided by: 
```
        _alt_xml_path = os.path.join(sys.prefix, 'share',
                                         'udunits', 'udunits2.xml')
```
I had to add 'Library' before 'share':
```
        _alt_xml_path = os.path.join(sys.prefix, 'Library', 'share',
                                         'udunits', 'udunits2.xml')
```

The second even resolving to the right directory didn't work.  This might be too 'hacky' for cf_units but It would be possible to add the working directory switch directly to the __init__.py so windows users could continue to use this install without grief.  This works for me:
![image](https://cloud.githubusercontent.com/assets/2249796/14437062/c9f54186-ffdc-11e5-8b11-17cb63054349.png)
